Names-values.Rmd

```diff
@@ -738,7 +738,7 @@ This number won't agree with the amount of memory reported by your operating sys
 1. R counts the memory occupied by objects but there may be empty gaps due to 
    deleted objects. This problem is known as memory fragmentation.
 
-## Answers {#names-values-answers}
+## Quiz Answers {#names-values-answers}
```

Comment:
I just realised this should be "Quiz answers" because I use sentence case for heading. Can you please do a PR to fix them all?

Reply:
Yes, we'll do it with the global changes after all the PR are resolved!